### DISCUSSION TITLE
Can`t add maintainers while creating a team with GitHub API

### DISCUSSION THREAD

**Discussion by OlexiyVasylenkoDev** [community/community#60575] [Apps, API and Webhooks] [answered]
### Select Topic Area

Bug

### Body

I am trying to create a team in GitHub organisation using GitHub API. The problem is that I can`t add maintainers to the team. The official documentation says:

```
maintainers array of strings
List GitHub IDs for organization members who will become team maintainers.
```
I am using Python and here is how I pass maintainers in request body: ['1111111'], where "1111111" is the future maintainer ID. However, I get this Response, which doesn`t any create a team for me.

```
{'message': 'Validation Failed', 'errors': [{'resource': 'Team', 'code': 'unprocessable', 'field': 'd
ata', 'message': 'Invalid maintainers were specified.'}], 'documentation_url': 'https://docs.github.c
om/rest/reference/teams#create-a-team'}
```

Unfortunately, I couldn`t find any information on this error in the oficial documentation. Maybe somebody has already faced this problem? Or maybe you have any suggestions on how to solve it? I would be very grateful for any advice!

Here is the function, from which I send the request:

```
def create_team(self, name, maintainers):
    url = self.ORGANIZATION_URL + "/teams"
    body = {
        "name": name,
        "maintainers": maintainers
    }
    headers = {
        "Authorization": f"Bearer {os.environ.get('GITHUB_TOKEN')}"
    }
    requests.post(url, json=body, headers=headers)
```

Token is not expired and the url is correct as well. This request works if I remove "maintainers" from request body.

[Official documentation on creating a team](https://docs.github.com/en/rest/teams/teams?apiVersion=2022-11-28#create-a-team)

**Comment by fabianf92** [comment, marked as answer]
I had the same issue, it worked for me when using the users login (in my case fabianf92), not the users ID

**Reply by OlexiyVasylenkoDev** [reply]
Thanks a lot! It resolved my problem
